$NetBSD: patch-aa,v 1.1 2004/11/30 12:08:52 wiz Exp $

Patch from
	http://rt.cpan.org/NoAuth/Bug.html?id=2244
with an additional fix (if .F match is first, it causes problem in timestamp
replacement).

--- t/try_examples.t.orig	2001-12-11 16:17:11.000000000 +0100
+++ t/try_examples.t
@@ -91,14 +91,14 @@ foreach my $e (@examples) {
 	    $_ .= "\n" unless /\n$/;
 	    # bin the naughty carriage returns
 	    s/\r//g;
+	    # hide things that look like CPU usage
+	    s{Time:\s+[\d\.]+\s+wallclock secs \([\d\s\.]+usr\s+\+[\d\s\.]+sys\s+=[\d\s\.]+CPU\)}
+	    {TIME-SUMMARY}g;
 	    # we can't assume the order of tests will be the same
-	    s/^[\.F]+$/TEST-RUN-SUMMARY/sm;
+	    s/^[.F]+/TEST-RUN-SUMMARY/sm;
 	    s/::Load[0-9_]+Anonymous[0-9_]+/::LOAD_ANONYMOUS_CLASSNAME/;
 	    # indent lines with '# ' so they're comments if the test fails
 	    s/\n/\n# /g;
-	    # hide things that look like CPU usage
-	    s{Time:\s+[\d\.]+\s+wallclock secs \([\d\s\.]+usr\s+\+[\d\s\.]+sys\s+=[\d\s\.]+CPU\)}
-	    {TIME-SUMMARY}g;
 	}
 	ok($out, $guru_checked{$e});
     } else {
